packages/material/scss/window/_layout.scss

```diff
@@ -1 +1,34 @@
 @import "~@progress/kendo-theme-default/scss/window/layout";
+
+@include exports("window/layout/material") {
+
+    $window-title-font-size: 20px !default;
+    $window-title-line-height: 1.4 !default;
+
+    .k-window-titlebar {
+        border-width: 0;
+    }
+    .k-window-title {
+        margin: -.2em 0;
```

Comment:
In the material guidelines, padding refers to the space between the edge of the rendered and surrounding containers. In essence, it's like having line-height 1, but with the minor difference that when title is multi-line, the line height is 28px.

Thus, I've resorted to some trickery to almost get the appearance right.